### PR TITLE
oc_volume: Add missing parameter documentation

### PR DESCRIPTION
**Pull request by jarrpa**
Signed-off-by: Jose A. Rivera <jarrpa@redhat.com>

### CODE REVIEW

File: roles/lib_openshift/library/oc_volume.py [hunk 1]
```diff
@@ -80,6 +80,18 @@
     required: false
     default: False
     aliases: []
+  name:
```
Comment (ashcrow) [inline roles/lib_openshift/library/oc_volume.py]: Isn't this ``resource_name``? I know ``name`` is used once the value is passed in but the initializer looks to me like ``resource_name``.

Comment (jarrpa) [inline roles/lib_openshift/library/oc_volume.py]: I was going based off of this: https://github.com/openshift/openshift-ansible/blob/master/roles/lib_openshift/src/class/oc_volume.py#L107

Comment (ashcrow) [inline roles/lib_openshift/library/oc_volume.py]: I see. It's a little confusing to be honest but it's not a deal breaker.

**Comment by ashcrow** [comment]
[test]

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
[merge]

**Pull request merged**